src/Avalonia.Controls/TopLevel.cs

```diff
@@ -75,6 +76,22 @@ public abstract class TopLevel : ContentControl,
         public static readonly StyledProperty<IBrush> TransparencyBackgroundFallbackProperty =
             AvaloniaProperty.Register<TopLevel, IBrush>(nameof(TransparencyBackgroundFallback), Brushes.White);
 
+        /// <summary>
+        /// Defines the StatusBarColor attached property.
+        /// </summary>
+        public static readonly AttachedProperty<Color?> StatusBarColorProperty =
```

Comment:
we can put this in its own file:

`<UserControl MobileStatusBar.Color="Red" />`